models/metrics.go

```diff
@@ -730,3 +730,24 @@ func (m *Metrics) SelectObjectDetailsLabels(ctx context.Context, periodStartFrom
 
 	return &res, nil
 }
+
+const fingerprintByQueryID = `SELECT fingerprint FROM metrics WHERE queryid = ? LIMIT 1`
+
+// GetFingerprintByQueryID returns the query fingerprint, used in query details.
+func (m *Metrics) GetFingerprintByQueryID(ctx context.Context, queryID string) (string, error) {
+	queryCtx, cancel := context.WithTimeout(ctx, queryTimeout)
+	defer cancel()
+
+	rows, err := m.db.QueryxContext(queryCtx, fingerprintByQueryID, []interface{}{queryID}...)
```

Comment:
```suggestion
	err := m.db.GetContext(queryCtx, &fingerprint, fingerprintByQueryID, queryID)
```
I think, for brevity better to use `GetContext`.